bin/fuel-core/src/cli/run.rs

```diff
@@ -193,6 +201,9 @@ pub struct Command {
     /// Time to wait after submitting a query before debug info will be logged about query.
     #[clap(long = "query-log-threshold-time", default_value = "2s", env)]
     pub query_log_threshold_time: humantime::Duration,
+
+    #[clap(flatten)]
+    pub profiling: profiling::ProfilingArgs,
```

Comment:
For the sake of consistency, I could understand removing the cfg, like we've done elsewhere. This way, you don't have conditional CLI args. This would be better than situation where you have added the args, but the associated behaviour is removed due to the compilation configuration.

Reply:
I removed the feature flag - the only downside is that this may increase compile times for those installing fuel-core for local dev testing.

Reply:
I think that's a reasonable tradeoff for more consistent and predictable behaviour.